frontends/web/src/components/forms/radio.module.css

```diff
@@ -50,7 +50,7 @@
 /* checked */
 
 .radio input:checked + label::before {
-    background-color: var(--color-white);
+    background-color: var(--background-secondary);
 }
 
```

Comment:
and at the end, I am not sure where disabled radio inputs are used.

```
.radio input:disabled + label::before {
    background: rgb(248, 248, 248);
    border-color: rgb(220, 220, 220);
```

Reply:
Same as comment above.